examples/TableWithCustomToolbar/TableWithCustomToolbar.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { Story, Meta } from '@storybook/react'
+import { Column } from 'react-table'
+
+import { DataTable, DataTableProps } from '../../src/DataTable'
+import { CustomToolbar } from './CustomToolbar'
+
+export default {
+  title: 'examples/TableWithCustomToolbar',
+  component: DataTable,
+} as Meta;
+
+type Person = {
+  firstName: string
+  lastName: string
+  age: number
+}
+
+type PersonData = Person & {
+  subRows?: PersonData[]
+}
+
+const data = [
+  {
+    firstName: 'Bill',
+    lastName: 'Murray',
+    age: 25,
+  },
+  {
+    firstName: 'Dan',
+    lastName: 'Aykroyd',
+    age: 25
+  }
+]
+
+const defaultItem = {
+  firstName: null,
+  lastName: null,
+  age: null,
+}
+
+const columns: Column<PersonData>[] = [
+  {
+    Header: 'First Name',
+    accessor: 'firstName'
+  },
+  {
+    Header: 'Last Name',
+    accessor: 'lastName'
+  },
+  {
+    Header: 'Age',
+    accessor: 'age'
+  }
+]
+
+const Template: Story<DataTableProps<PersonData>> = (args) => {
```

Comment:
@Emmaalara this example was using external state to simulate appending data to the table. We shouldn't need it for this example. Would you use the default component as we are in the custom row example?